twilight-gateway/src/shard.rs

```diff
@@ -893,18 +894,17 @@ impl Shard {
             Some(OpCode::Hello) => {
                 let event = Self::parse_event::<Hello>(buffer)?;
                 let heartbeat_interval = Duration::from_millis(event.data.heartbeat_interval);
+                // First heartbeat should have some jitter, see
+                // https://discord.com/developers/docs/topics/gateway#heartbeat-interval
+                let jitter = heartbeat_interval.mul_f64(rand::random());
```

Comment:
I think this should be moved into its own pr

Reply:
Jittering was already included (see `start` variable)

Reply:
Ahh yeah my mistake